### ISSUE TITLE
Builds on unsupported systems

### ISSUE THREAD

**Issue by kamadorueda** [garnix-io/issues#5]
Log: https://garnix.io/build/kamadorueda/alejandra/511c3f6a88b6964e1496fb6f441f4ae5e58bd3ea/alejandra-x86_64-apple-darwin

```
repo: kamadorueda/alejandra
commit: 511c3f6a88b6964e1496fb6f441f4ae5e58bd3ea
package: alejandra-x86_64-apple-darwin
status: Failure
started at: 2022-03-04T02:29:36.266444Z
finished at: 2022-03-04T02:29:36.8271Z

Logs
No logs available
```

Garnix is creating builds on GitHub for unsupported systems:

![image](https://user-images.githubusercontent.com/47480384/156689209-223f19d1-da11-4e19-af65-d980b708489c.png)

As a user I would expect to either:
- Have all my packages built, at least for the systems supported by hydra.nixos.org: 
  ```
  nix-repl> lib.systems.supported.hydra
  [ "x86_64-linux" "aarch64-linux" "x86_64-darwin" "armv6l-linux" "armv7l-linux" "i686-linux" "mipsel-linux" "aarch64-darwin" ]
  ```
  
  or at least tier1 and tier2:
  ```
  nix-repl> lib.systems.supported.tier1
  [ "x86_64-linux" ]

  nix-repl> lib.systems.supported.tier2
  [ "aarch64-linux" "x86_64-darwin" ]
  ```
- Hide the builds for unsupported systems
- Show the builds as another status than failed, maybe skipped?

**Comment by jkarni** [comment]
I just pushed some changes that no longer includes builds from other architectures. (See [here](https://github.com/jkarni/alejandra/runs/5428322288) for the same run.) 

aarch64-linux and aarch64-darwin are planned, but I'll open issues here for them as well. I'm inclined to think that, nix-official tiers notwithstanding, aarch64-darwin is a wiser investment than x86_64-darwin?

Closing, but feel free the reopen if you feel that was a mistake!

(Thanks for the clear and helpful reports!)
